ui/org.eclipse.pde.junit.runtime/readme.txt

```diff
@@ -0,0 +1,18 @@
+ This PDE JUnit runtime is added to test-runtimes launched from an Eclipse workspace and allows JUnit tests to be run with an OSGi runtime.
```

Comment:
It would be nice to make this README.md so it's auto rendered by github when viewing this folder.

Reply:
Good suggestion. Done.